wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/DeferredCommand.java

```diff
@@ -28,9 +28,11 @@ public class DeferredCommand extends Command {
   private Command m_command = m_nullCommand;
 
   /**
-   * Creates a new DeferredCommand that runs the supplied command when initialized, and ends when it
-   * ends. Useful for lazily creating commands at runtime. The {@link Supplier} will be called each
-   * time this command is initialized. The Supplier <i>must</i> create a new Command each call.
+   * Creates a new DeferredCommand that directly runs the supplied command when initialized, and
+   * ends when it ends. Useful for lazily creating commands when the DeferredCommand is reached in a
+   * composition, such as if the supplied command depends on runtime state. The {@link Supplier}
```

Comment:
Is lazily creating commands only applicable within compositions? Wouldn't that also be useful in button bindings? (e.g., OTF path following)

Reply:
That was an attempt to indicate that it's at command init (which is different from construction time), maybe this is just a "go read the docs" moment?